src/new-client/src/components/DraggableTearOut/DraggableTearOut.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from "styled-components"
+import { RefObject, useContext, useState } from "react"
+import { TearOutContext } from "@/App/TearOutSection/tearOutContext"
+import { tearOutSection, Section } from "@/App/TearOutSection/state"
+import { canDrag } from "@/components/DraggableTearOut/canDrag"
+import { tearOut } from "@/App/LiveRates/Tile/TearOut/state"
+
+export const DragWrapper = styled.div`
+  height: 100%;
+`
+interface DraggableTearOutProps {
+  children: JSX.Element[] | JSX.Element
+  section: Section
+  isTile?: boolean
+  ref?: RefObject<HTMLDivElement>
+  symbol?: string
+  disabled?: boolean
+}
+
+export const DraggableTearOut: React.FC<DraggableTearOutProps> = (
```

Comment:
Some further thoughts - 
I'm thinking it might be nice to have 2 different components - `DraggableSectionTearOut` and `DraggableTileTearOut`  because at the moment there is nothing stopping you saying `isTile: true` without passing a `symbol`, and `section` is mandatory when it is not needed when used by tiles.
They can both use the same `DraggableTearOut` component but have different Props, and `DraggableTearOut` can be simplified.

EG:
```
type DraggableSectionTearOutProps = React.PropsWithChildren<{
  section: Section;
  disabled?: boolean;
}>

type DraggableTileTearOutProps = React.PropsWithChildren<{
  symbol: string;
  tileRef: RefObject<HTMLDivElement>;
  disabled? boolean;
}>

type DraggableTearOutProps = React.PropsWithChildren<{
  dragHandler: () => void;
  // could pass disabled down here, but I think it might be better to just return the children in the parent component instead of rendering DraggableTearOut at all
}>
```

Reply:
Fixed at `DraggableTearout.tsx`